lang/ru.xml

```diff
@@ -64,32 +64,65 @@
 			<day>суббота</day>
 		</wday>
 		<ymonth>
-			<month>Январь</month>
-			<month>Февраль</month>
-			<month>Март</month>
-			<month>Апрель</month>
-			<month>Май</month>
-			<month>Июнь</month>
-			<month>Июль</month>
-			<month>Август</month>
-			<month>Сентябрь</month>
-			<month>Октябрь</month>
-			<month>Ноябрь</month>
-			<month>Декабрь</month>
+			<month>Январь,</month>
+			<month>Февраль,</month>
+			<month>Март,</month>
+			<month>Апрель,</month>
+			<month>Май,</month>
+			<month>Июнь,</month>
+			<month>Июль,</month>
+			<month>Август,</month>
+			<month>Сентябрь,</month>
+			<month>Октябрь,</month>
+			<month>Ноябрь,</month>
+			<month>Декабрь,</month>
 		</ymonth>
+		<mday>
+			<days>31</days>
+			<days>1</days>
+			<days>2</days>
+			<days>3</days>
+			<days>4</days>
+			<days>5</days>
+			<days>6</days>
+			<days>7</days>
+			<days>8</days>
+			<days>9</days>
+			<days>10</days>
+			<days>11</days>
+			<days>12</days>
+			<days>13</days>
+			<days>14</days>
+			<days>15</days>
+			<days>16</days>
+			<days>17</days>
+			<days>18</days>
+			<days>19</days>
+			<days>20</days>
+			<days>21</days>
+			<days>22</days>
+			<days>23</days>
+			<days>24</days>
+			<days>25</days>
+			<days>26</days>
+			<days>27</days>
+			<days>28</days>
+			<days>29</days>
+			<days>30</days>
+		</mday>
 		<small_ymonth>
-			<month>января</month>
-			<month>февраля</month>
-			<month>марта</month>
-			<month>апреля</month>
-			<month>мая</month>
-			<month>июня</month>
-			<month>июля</month>
-			<month>августа</month>
-			<month>сентября</month>
-			<month>октября</month>
-			<month>ноября</month>
-			<month>декабря</month>
+			<month>января,</month>
+			<month>февраля,</month>
+			<month>марта,</month>
+			<month>апреля,</month>
+			<month>мая,</month>
+			<month>июня,</month>
+			<month>июля,</month>
+			<month>августа,</month>
+			<month>сентября,</month>
+			<month>октября,</month>
+			<month>ноября,</month>
+			<month>декабря,</month>
```

Comment:
why russian have , ? you have screen for it ? 🤔

Reply:
Wait a minute, Mr. Zangetsu. i took a screenshot in my PSVita.

Reply:
Your screenshots no show ,  in end month ? 🤔

Reply:
eu you can check if other disposition havbe this , ? have 3 mode for that

Reply:
i speek of SCE_SYSTEM_PARAM_DATE_FORMAT

Reply:
> why russian have , ? you have screen for it ? 🤔

ohhh, i found it, it is my wrong.

Reply:
> i speek of SCE_SYSTEM_PARAM_DATE_FORMAT

I really appreciate Mr. Zangetsu that I didn't realize I made such a small mistake.

Reply:
Sorry, i have no view, but so, i asked if have , for motn aon 3 mode of date format ?

Reply:
> Sorry, i have no view, but so, i asked if have , for motn aon 3 mode of date format ?

is none, i didn't notice it the first time.

Reply:
> > Sorry, i have no view, but so, i asked if have , for motn aon 3 mode of date format ?
> 
> is none, i didn't notice it the first time.

is what ?

Reply:
> is what ?

the ","

Reply:
you lost me here 🤔 you have pushed bu code in not changed,for both you change caps month and yes on your psvita screen caps month have , in end

Reply:
> you lost me here 🤔 you have pushed bu code in not changed,for both you change caps month and yes on your psvita screen caps month have , in end

thx u so mush, Mr. Zangetsu, u help me a lot. 😭